databricks/koala/dask/utils.py

```diff
@@ -62,8 +62,8 @@ def derived_from(original_klass, version=None, ua_args=[]):
     version : str
         Original package version which supports the wrapped method
     ua_args : list
-        List of keywords which Dask doesn't support. Keywords existing in
-        original but not in Dask will automatically be added.
+        List of keywords which Koala doesn't support. Keywords existing in
```

Comment:
I forgot. Let's keep the dask code as close as possible to the original code. This will help when trying to backport upstream fixes.